posts/Guillaume/2017-07-11-hibernate-validator-600-cr3-out.adoc

```diff
@@ -0,0 +1,41 @@
+= "Hibernate Validator 6.0.0.CR3 is out with Bean Validation 2.0.0.CR3 support"
+Guillaume Smet
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+---
+We just released the Bean Validation 2.0 release train (e.g. the specification, the API and the TCK) for the Final Approval Ballot and, as usual, we release a compatible version of Hibernate Validator shortly after: here comes Hibernate Validator 6.0.0.CR3.
+
+== What's new since CR2
+
+As the final release of Bean Validation is getting closer, we mainly focused on keeping up with the latest spec clarifications and improving a few things here and there.
+
+Here are the highlights of this release:
+
+ * We removed the support for constraints and cascading on container elements of arrays. Basically, it was not possible to be backward compatible with Bean Validation 1.1 and introduce this support, considering how array annotations are handled according to the JLS. As we don't think there's much traction for it, we stayed on the safe side for now.
```

Comment:
I'd add that it has been decided against it on the spec level and add a link to the discussion on the mailing list.